pkg/report/linux.go

```diff
@@ -376,7 +376,14 @@ func (ctx *linux) Symbolize(rep *Report) error {
 		}
 	}
 
+	oldLen := len(rep.Report)
 	rep.Report = ctx.decompileOpcodes(rep.Report, rep)
+	if len(rep.Report) > 0 && rep.reportPrefixLen > len(rep.Report) {
+		// An attempt to catch #4198.
+		panic(fmt.Sprintf("invalid reportPrefixLen (%d) after decompileOpcodes, report len: %d -> %d",
```

Comment:
I would print raw reports we parsed as %q.
Numbers won't necessary answer how/why we get to this broken state.

Reply:
Added printing the whole `Report` object as `%+v`.